.github/workflows/tests.yml

```diff
@@ -98,3 +97,57 @@ jobs:
     - name: Test with pytest
       run: |
         pytest -v ./tests/sealapi
+  
+  SEAL_API_WIN:
+
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+          
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements_dev.txt
+    - name: Build the library
+      run: |
+        # fetch third_party libraries
+        git submodule init && git submodule update
+        # we need to install tenseal in order to build the cpp library
+        pip install .                
+    - name: Test with pytest
+      run: |
+        pytest -v ./tests/sealapi
```

Comment:
Don't expect to setup Windows the same way as Linux, checkout this https://github.com/OpenMined/TenSEAL/blob/master/.github/workflows/pythonpublish-windows.yml

Reply:
ok